src/styles/stopwatch.scss

```diff
@@ -0,0 +1,73 @@
+.stopwatch {
+  position: absolute;
+  top: 0;
+  right: 0;
+  left: 0;
+  bottom: 0;
+  margin: auto;
+  background-color: #ffd700;
+  border: 4px #daa520;
+  border-color: #daa520;
+  box-shadow: 10px;
```

Comment:
it doesn't work
![image](https://user-images.githubusercontent.com/86923089/215333660-d581e3f9-0709-4e3a-9b27-6e853db8d985.png)